modules/rostests/apitests/ntdll/NtQueryInformationProcess.c

```diff
@@ -185,6 +185,41 @@ Test_ProcessTimes(void)
 #undef SPIN_TIME
 }
 
+static
+void
+Test_ProcessPriorityClass(void)
+{
+    NTSTATUS Status;
+    PROCESS_PRIORITY_CLASS ProcPriority;
```

Comment:
I am not sure of alignment of this, since it's on the stack.
Better would be to allocate it with malloc, or HeapAlloc.